recon/core/playbook.py

```
@@ -0,0 +1,150 @@
+    "leaks": "ergenekon-leak",
+    "leak": "ergenekon-leak",
+}
+
+
+def _format_args(template_args: Dict[str, str], params: Dict[str, Any]) -> Dict[str, Any]:
+    out: Dict[str, Any] = {}
+    for k, v in template_args.items():
+        if isinstance(v, str):
+            try:
+                out[k] = v.format(**params)
+            except Exception:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)